app/containers/NewMediaCall/NewMediaCall.integration.test.tsx

```
@@ -0,0 +1,407 @@
+// ─── Tests ────────────────────────────────────────────────────────────────────
+
+describe('NewMediaCall → CallView (integration)', () => {
+	beforeEach(() => {
+		jest.clearAllMocks();
+		createdSessions.length = 0;
+		usePeerAutocompleteStore.getState().reset();
+		useCallStore.getState().reset();
+		mediaSessionInstance.reset();
+		consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+		mediaSessionInstance.init('me');
+		expect(createdSessions).toHaveLength(1); // singleton-bleed guard
```

ESLint

[error] Expect must be inside of a test block

(jest/no-standalone-expect)